forward_command_controller/forward_command_plugin.xml

```diff
@@ -4,4 +4,10 @@
     The forward command controller commands a group of joints in a given interface
   </description>
   </class>
+  <class name="forward_command_controller/MultiInterfaceForwardController"
+         type="forward_command_controller::MultiInterfaceForwardController" base_class_type="controller_interface::ControllerInterface">
+    <description>
+      MultiInterfaceForwardController ros2_control controller.
```

Comment:
A better description would be good here. We should make it clear that this controller is for some very specific use-cases and in general recommend ppl to use the single interface version to avoid mixing different units in the (essentially) untyped input ROS message.

Reply:
Yeah... This controller could become a basis for GPIO controller actually. We could discuss this in the WG-meeting.